README.md

```diff
@@ -53,7 +53,7 @@ https://hub.docker.com/r/fallenbagel/jellyseerr
 
 Pre-requisites:
 
-- Nodejs [v18](https://nodejs.org/download/release/v18.18.2)
```

Comment:
Not a typo.

Reply:
@Gauvino this is not a typo. Please do not change the versions.

Reply:
Removed changed !